### PR TITLE
Some css fixes

### PR DESCRIPTION
**Pull request by Farahkh**
fixed social media links are opening in the same page. fixed speaker's social media is showing with empty link.
please let me know if the deploy successful.

### CODE REVIEW

File: lib/pages/flutteristas_conference.dart
Comment (Stef-GMS) [inline lib/pages/flutteristas_conference.dart:1]: Do you mean have have this not use the `dev` on the dev server?

**Review by Stef-GMS — COMMENTED** [review]
Do you mean have have this not use the dev on the dev server?

File: lib/pages/flutteristas_conference.dart
Comment (Farahkh) [reply lib/pages/flutteristas_conference.dart:1]: yes, I changed it to use the production database so Khushbu can start updating straight there...

**Review by Stef-GMS — APPROVED** [review]
Approved

**Pull request merged**